StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OnLoadErrorMessageHandler.swift

```diff
@@ -7,6 +7,13 @@
 
 import Foundation
 
+extension OnLoadErrorMessageHandler.Values.ErrorValue {
+    var connectEmbedError: EmbeddedComponentError {
+        // API Error is a catch all so defer to that if we get an unknown type.
```

Comment:
We also plan to log an error if the type is unrecognized
https://docs.google.com/document/d/1nzHGofoclzij4nP5n5qgSpYLia-93heStXvt7vOfrkQ/edit?pli=1#bookmark=id.fd4twivivcoi

```suggestion
        // API Error is a catch all so defer to that if we get an unknown type.
        // TODO(MXMOBILE-2491): Log error analytic if `type` is unrecognized
```